### PR TITLE
Int 1346 field in demographic info section of the person card are not align

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- [Field in demographic info section of the person card are not align INT-1346](https://osi-cwds.atlassian.net/browse/INT-1346)

## Description
Fixed the fields alignment in the demographic info section.
Fixed approximate age label alignment when the window is resized smaller.
I have attached the screenshot in JIRA. 

## Tests
No behavioral change so test not needed.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by pokharelmanish** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

File: app/assets/stylesheets/form.scss [hunk 1]
```diff
@@ -149,15 +149,6 @@ input[type='radio']:disabled + label::before {
   cursor: not-allowed;
 }
 
-.input-no-header {
```
Comment (wmitchell-cambria) [inline app/assets/stylesheets/form.scss]: These classes aren't used anywhere else, are they?

Comment (TylerOSI) [inline app/assets/stylesheets/form.scss]: is this class being used anywhere else?

File: app/assets/stylesheets/form.scss [hunk 2]
```diff
@@ -149,15 +149,6 @@ input[type='radio']:disabled + label::before {
...
-  margin-top: 56px;
-}
-
-.text-between-inputs {
```
Comment (TylerOSI) [inline app/assets/stylesheets/form.scss]: same: is this class being used anywhere else?

File: app/assets/stylesheets/form.scss (hunk 1 above)
Comment (pokharelmanish) [reply app/assets/stylesheets/form.scss]: No

File: app/assets/stylesheets/form.scss (hunk 2 above)
Comment (pokharelmanish) [reply app/assets/stylesheets/form.scss]: No

File: app/assets/stylesheets/form.scss (hunk 1 above)
Comment (pokharelmanish) [reply app/assets/stylesheets/form.scss]: No.

File: app/javascript/views/people/PersonDemographicsForm.jsx [hunk 3]
```diff
@@ -33,9 +33,9 @@ const PersonDemographicsForm = ({
         value={dateOfBirth}
         onChange={(value) => onChange('date_of_birth', value)}
       />
-      <div className='col-md-1 text-between-inputs' >or</div>
+      <div className='col-md-1'><p><label>&nbsp;</label>or</p></div>
```
Comment (brunoOSI) [inline app/javascript/views/people/PersonDemographicsForm.jsx]: Really not a fan of `&nbsp;` here. 
I think you can achieve the right outcome just with CSS

**Comment by pokharelmanish** [comment]
retest this please

File: app/assets/stylesheets/form.scss (hunk 2 above)
Comment (brunoOSI) [reply app/assets/stylesheets/form.scss]: did you try adding a `padding-top:X` to this class ?

Comment (pokharelmanish) [reply app/assets/stylesheets/form.scss]: Yes, I had tried but if we use this we need to hardcode the value.

**Comment by pokharelmanish** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

File: app/javascript/views/people/PersonDemographicsForm.jsx [hunk 4]
```diff
@@ -44,7 +44,7 @@ const PersonDemographicsForm = ({
         onChange={({target: {value}}) => onChange('approximate_age', value)}
         disabled={approximateAgeIsDisabled}
       />
-      <div className='col-md-2 input-no-header'>
+      <div className='col-md-2 text-between-inputs'>
```
Comment (brunoOSI) [inline app/javascript/views/people/PersonDemographicsForm.jsx]: This is not text between inputs, as the previous class stated clearly, it is an input without a label. I would revert any markup changes (including the one above `<p>or</p>`). 
Keep the changes on CSS only.

Comment (pokharelmanish) [reply app/javascript/views/people/PersonDemographicsForm.jsx]: ok, will do.

**Comment by pokharelmanish** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

**Pull request merged**